src/Microsoft.Framework.PackageManager/Bundle/BundleProject.cs

```diff
@@ -507,5 +501,63 @@ private string BasePath(string relativePath)
             var index2 = (relativePath + Path.AltDirectorySeparatorChar).IndexOf(Path.AltDirectorySeparatorChar);
             return relativePath.Substring(0, Math.Min(index1, index2));
         }
+
+        private Runtime.Project ResolveProject(string name)
+        {
+            Runtime.Project project;
+            if (!_projectResolver.TryResolveProject(name, out project))
+            {
+                throw new Exception("TODO: unable to resolve project named " + name);
+            }
+            return project;
+        }
+
+        private bool IsWrappingAssembly()
+        {
+            var project = ResolveProject(_libraryDescription.Identity.Name);
+
+            /* If this project is wrapping an assembly, the project.json has the format like:
+            {
+              "frameworks": {
+                "net45": {
+                  "bin": {
+                    "assembly": "relative/path/to/ClassLibrary1.dll"
+                  }
+                }
+              }
+            } */
+            var rootObj = JObject.Parse(File.ReadAllText(project.ProjectFilePath));
+            var frameworksObj = rootObj["frameworks"] as JObject;
+
+            if (frameworksObj == null)
+            {
+                return false;
+            }
+
+            foreach (var frameworkProp in frameworksObj.Properties())
```

Comment:
This loop executes once at most in all cases (there is no break/continue in it and return is the last statement)... Why is it a loop?